### PR TITLE
ICU-21901 Update build/deploy for icu4jweb

### PR DESCRIPTION
**Pull request by sven-oly**

### CODE REVIEW

File: icu4jweb/README.md [hunk 1]
```diff
@@ -2,13 +2,62 @@
...
 # RUNNING WITH [docker](https://docker.io)
+Note that this does not rebuild the demos, but just creates a new docker image for running locally.
 
 - `sh build-docker.sh`
 - `docker run --rm -p 8083:8080 unicode/icu4jweb:latest`  (^C to stop and delete the container)
-- now go to [http://127.0.0.1:8083/icu4jweb/](http://127.0.0.1:8083/icu4jweb/)
+- Verify that each demo program works [Run latest docker image](http://127.0.0.1:8083/icu4jweb/).
+
+# Deploy to [public demo](https://icu4j-demos.unicode.org/icu4jweb/)
+To publish the demos, one needs access to the Google Cloud [ICU4C demos project](https://pantheon.corp.google.com/run/deploy/us-central1/icu4jweb?project=icu4c-demos).
```
Comment (srl295) [inline icu4jweb/README.md:23]: I still don't know what pantheon is.  It should be https://console.cloud.google.com to not use an internal Google URL.
```suggestion
To publish the demos, one needs access to the Google Cloud [ICU4C demos project](https://console.cloud.google.com/run/deploy/us-central1/icu4jweb?project=icu4c-demos).
```

File: icu4jweb/README.md [hunk 2]
```diff
@@ -2,13 +2,62 @@
...
+- `gcloud auth configure-docker`
+- Edit push-deploy.sh, replacing "latest" with the tag used in build-deploy.sh.
+- Now push the tagged image to the GCloud:
+- `sh push-deploy.sh`
+- Verify that the push succeeds. Expect a command such as
+"71.1: digest: sha256:d00e3bad53511402d4f9376b01cb9c3274f1cf6e75281aabb75e1c6e3f8766e0 size: 1999"
+
+
+## Managing Docker images in Gcloud
+If necessary, use this link to rename, delete, and otherwise [mangage Docker images](https://pantheon.corp.google.com/gcr/images/icu4c-demos?project=icu4c-demos).
```
Comment (srl295) [inline icu4jweb/README.md:40]: same

File: icu4jweb/build-deploy.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+set -x
+# the project id "icu4c-demos" cannot be changed.
+# Update the final part of the tag for each new release.
+docker build -t gcr.io/icu4c-demos/icu4jweb:71.1 .
```
Comment (srl295) [inline icu4jweb/build-deploy.sh:5]: future suggestion, maybe take the ICU version from the command line

**Review by srl295 — APPROVED** [review]
I'd fix the internal URLs. Not sure if the images are working now or not, but this is an improvement…

File: icu4jweb/README.md (hunk 2 above)
Comment (sven-oly) [reply icu4jweb/README.md]: Thanks for the comments. Updated

Comment (sven-oly) [reply icu4jweb/README.md]: Images are now working.

File: icu4jweb/build-deploy.sh (hunk 3 above)
Comment (sven-oly) [reply icu4jweb/build-deploy.sh:5]: I'll note that.

**Review by srl295 — APPROVED** [review]

**Review by markusicu — APPROVED** [review]
rslgtm

please *squash*-and-merge

**Pull request merged**